test/pirls.jl

```diff
@@ -91,18 +91,25 @@ end
     @testset "GLMM refit" begin
         gm2r = deepcopy(gm2)
         @test_throws ArgumentError fit!(gm2r)
-        refit!(gm2r, 1 .- gm2.y; fast=true)
-        @test gm2r.β ≈ -gm2.β atol=1e-3
+
+        refit!(gm2r; fast=true, progress=false)
+        @test length(gm2r.optsum.final) == 1
         @test gm2r.θ ≈ gm2.θ atol=1e-3
 
-        refit!(gm2r, 1 .- gm2.y; fast=false)
+        # swapping successes and failures to give us the same model
+        # but with opposite signs. healthy ≈ 1 - response(gm2r)
+        # but defining it in terms of the original values avoids some
+        # nasty floating point issues
+        healthy = @. (cbpp.hsz - cbpp.incid) / cbpp.hsz
+        refit!(gm2r, healthy; fast=false, progress=false)
+        @test length(gm2r.optsum.final) == 5
```

Comment:
@dmbates going the `kwargs...` route made me realize that we weren't passing all the arguments here ... which then lead to the discovery that the "flipped" response was numerically pathological for fast fits. Hence the changes here.